module/solutions/helper.js

```diff
@@ -1026,6 +1057,9 @@ module.exports = class SolutionsHelper {
 					} else {
 						userRoleKeys.forEach((key) => {
 							let scope = 'scope.' + key
+							if (prefix != '') {
```

Comment:
@Prajwal17Tunerlabs what will be the value of prefix?

Reply:
to form query im passing a prefix in the function calling. i want query to be formed as "acl.scope..." so in this case prefix  = 'acl'.